app/views/session_timeout/_ping.js.erb

```diff
@@ -42,7 +43,7 @@ function success(data) {
       show_warning = time_remaining < warning
 
   if (!data.live) {
-    window.LoginGov.autoLogout();
+    window.LoginGov.autoLogout(timeout_path);
```

Comment:
You named the variable above `timeoutPath`.

```suggestion
    window.LoginGov.autoLogout(timeoutPath);
```

Reply:
thank you for catching this 😐